keyboards/polykybd/tools/status_oled42_preview.py

```
@@ -180,43 +180,47 @@ def draw_bitmap(setpix, data, ox, oy, w=16, h=16):
+
+
+def build(side, mid, tiny, icons, world, contrast=35, layout_name=SHORT_NAMES[0],
+          lang='en-US', wpm=0):
     """side 'L' (USB host, brightness+WPM) or 'R' (Link bridge, locks)."""
     pts = set()
-    setp = lambda px, py: pts.add((px, py))
+    def setp(px, py):
+        pts.add((px, py))
     # role
     if side == 'L':
         draw_bitmap(setp, USB_BMP, -3, 0); draw_text(setp, tiny, 10, 12, 'Usb')
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)